### DISCUSSION TITLE
cannot destructure property "isSubmitting" of "t.formik" as it is undefined

### DISCUSSION THREAD

**Discussion by gurkiratkn** [jaredpalmer/formik#4000] [Q&A]
Getting above error on page load where I used formik
![image (3)](https://github.com/user-attachments/assets/3fa5f099-c4ce-4fdd-8dce-0e30d158715e)

**Comment by versently** [comment]
Hi, can you show your code?

**Reply by gurkiratkn** [reply]
Hey! I can't share the code directly, but could you give me some guidance on where to start debugging and what areas to focus on to resolve this issue?
@versently Thanks
